src/Build.UnitTests/BackEnd/BinaryTranslator_Tests.cs

```diff
@@ -258,7 +258,7 @@ public static IEnumerable<object[]> GetBuildExceptionsAsTestData()
         [MemberData(nameof(GetBuildExceptionsAsTestData))]
         public void TestSerializationOfBuildExceptions(Type exceptionType)
         {
-            Exception e = (Exception)Activator.CreateInstance(exceptionType, BindingFlags.NonPublic | BindingFlags.Public | BindingFlags.CreateInstance | BindingFlags.Instance, null, new object[]{"msg", new GenericBuildTransferredException() }, System.Globalization.CultureInfo.CurrentCulture);
+            Exception e = (Exception)Activator.CreateInstance(exceptionType, BindingFlags.NonPublic | BindingFlags.Public | BindingFlags.CreateInstance | BindingFlags.Instance, null, new object[] { "msg", new GenericBuildTransferredException() }, System.Globalization.CultureInfo.CurrentCulture);
```

Comment:
```suggestion
            Exception e = (Exception)Activator.CreateInstance(
            exceptionType, 
            BindingFlags.NonPublic | BindingFlags.Public | BindingFlags.CreateInstance | BindingFlags.Instance,
            null, 
            new object[] { "msg", new GenericBuildTransferredException() },
            System.Globalization.CultureInfo.CurrentCulture);
```

Reply:
Is it ok if I do one more indent there to make the createInstance parameters more pronounced?:
```
Exception e = (Exception)Activator.CreateInstance(
    exceptionType,
    BindingFlags.NonPublic | BindingFlags.Public | BindingFlags.CreateInstance | BindingFlags.Instance,
    null,
    new object[] { "msg", new GenericBuildTransferredException() },
    System.Globalization.CultureInfo.CurrentCulture);
```

Reply:
Yeah, I think that looks better.